packages/app/src/app/overmind/onInitialize.ts

```diff
@@ -1,136 +0,0 @@
-import { Context } from '.';
-
-export const onInitializeOvermind = async (
```

Comment:
Oh wow, we don't use this anymore?

Reply:
Its been moved here `packages/app/src/app/overmind/actions.ts` in the latest release of overmind